pkg/controller/bootstrap/bootstrap.go

```diff
@@ -255,6 +255,19 @@ func (b *Bootstrap) Run(destDir string) error {
 			return err
 		}
 	}
+	buf := bytes.Buffer{}
+	err = encoder.Encode(cconfig, &buf)
+	if err != nil {
+		return err
+	}
+	cconfigDir := filepath.Join(destDir, "controller-config")
+	if err := os.MkdirAll(cconfigDir, 0o764); err != nil {
```

Comment:
confirming that directory being created with permissions and the file being written with permissions are correct! not sure how restrictive we want to be (if at all)